app/javascript/shared/components/__tests__/useKeyboardShortcuts.test.jsx

```diff
@@ -18,6 +18,54 @@ describe('Keyboard shortcuts for components', () => {
       expect(shortcut.KeyK).toHaveBeenCalledTimes(1);
     });
     
+    it('should fire a function when longer chained keydown is detected', async () => {
+      const shortcut = {
+        "KeyA~KeyB": jest.fn()
+      };
+
+      renderHook(() =>
+        useKeyboardShortcuts(shortcut, document, { timeout: 1500 }),
+      );
+      fireEvent.keyDown(document, { code: "KeyA" });
+      await new Promise(resolve => setTimeout(() => {
+        fireEvent.keyDown(document, { code: "KeyB" });
+        resolve();
+      },600));
+
+      expect(shortcut["KeyA~KeyB"]).toHaveBeenCalledTimes(1);
+    });
+    
+    it('should fire a function when chained keydown is detected', () => {
+      const shortcut = {
+        "KeyA~KeyB": jest.fn()
+      };
+
+      renderHook(() =>
+        useKeyboardShortcuts(shortcut, document),
+      );
+      fireEvent.keyDown(document, { code: "KeyA" });
+      fireEvent.keyDown(document, { code: "KeyB" });
+
+      expect(shortcut["KeyA~KeyB"]).toHaveBeenCalledTimes(1);
+    });
+    
+    it('should not fire a function when chained keydown is missed', async () => {
+      const shortcut = {
+        "KeyA~KeyB": jest.fn()
+      };
+
+      renderHook(() =>
+        useKeyboardShortcuts(shortcut, document),
+      );
+      fireEvent.keyDown(document, { code: "KeyA" });
+      await new Promise(resolve => setTimeout(() => {
+        fireEvent.keyDown(document, { code: "KeyB" });
+        resolve();
+      },600));
```

Comment:
We can use the `timeout` value above and add a minimal amount of time to get this test to pass.

```suggestion
      },timeout + 25));
```